### PR TITLE
fix: overview page loading state

### PR DESCRIPTION
**Pull request by raykyri**
When there are no threads in a community, the Overview page shows a loading spinner forever.

Also fixes an issue with the CWP community which was being initialized with an invalid chain.

## How has this been tested?

- Test CWP: https://localhost:8080/common-protocol
- Test an community with no threads
- Test a community with threads - ensure that loading spinner still works as expected

## Does this PR affect any server routes?
- [ ] yes
- [X] no

## Have proper tags been added (for bug, enhancement, breaking change)?
- [X] yes

### CODE REVIEW

File: packages/commonwealth/server/migrations/20221213192007-add-projects.js [hunk 1]
```diff
@@ -51,7 +51,7 @@ module.exports = {
             default_symbol: 'CWP',
             name: 'Common Protocol',
             type: 'dao',
-            network: 'common-protocol',
+            network: 'ethereum',
```
Comment (jnaviask) [inline packages/commonwealth/server/migrations/20221213192007-add-projects.js:54]: This change will affect Chain Events handling -- we should instead add a new loading case for common protocol.

**Review by jnaviask — CHANGES_REQUESTED** [review]
Good catch with the overview page -- but the migration change is not exactly what we want.

**Comment by jnaviask** [comment]
Going to undo the CWP change.

**Review by jnaviask — APPROVED** [review]

**Comment by raykyri** [comment]
LGTM

File: packages/commonwealth/server/migrations/20221213192007-add-projects.js (hunk 1 above)
Comment (raykyri) [reply packages/commonwealth/server/migrations/20221213192007-add-projects.js]: We could also remove it?

**Pull request merged**